pkg/operator/connectivitycheckcontroller/connectivity_check_controller.go

```diff
@@ -61,6 +66,11 @@ func (c *connectivityCheckController) WithPodNetworkConnectivityCheckFn(podNetwo
 	return c
 }
 
+type unsupportedConfigOverrides struct {
```

Comment:
godoc.  This is a partial struct you're using to deserialize part (but not all) of unsupportedconfigoverrides

Reply:
Done.